### PR TITLE
Fix search by sector

### PR DESCRIPTION
**Pull request by richtier**
[this task](https://uktrade.atlassian.net/browse/ED-1336)

### CODE REVIEW

File: company/helpers.py [hunk 1]
```diff
@@ -8,18 +8,27 @@
...
 
 logger = logging.getLogger(__name__)
 company_profile_url = 'https://api.companieshouse.gov.uk/company/{number}'
 
 companies_house_session = requests.Session()
 
 
+def get_sector_label(sectors_value):
+    if not sectors_value:
+        return sectors_value
```
Comment (konradko) [inline company/helpers.py]: Don't get this logic

Comment (richtier) [inline company/helpers.py]: its a copy and paste of some FAB logic. I'll simplify

File: company/migrations/0042_auto_20170505_1148.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+from __future__ import unicode_literals
+
+from django.core import management
+from django.db import migrations
+
+def add_company_search_index_and_populate(apps, schema_editor):
+    management.call_command('rebuild_elasticsearch_index')
+
+
+def noop(*args, **kwargs):
```
Comment (konradko) [inline company/migrations/0042_auto_20170505_1148.py]: is this needed?

File: company/search.py [hunk 3]
```diff
@@ -29,7 +31,8 @@ class CompanyDocType(DocType):
     modified = FormattedDate(date_format='%Y-%m-%dT%H:%M:%S.%fZ')
     name = field.Text()
     number = field.Text()
-    sectors = field.Text()
+    sectors = field.Text(multi=True)
+    sectors_label = field.Text(multi=True)
```
Comment (konradko) [inline company/search.py]: maybe `sectors_labelled`?

File: company/migrations/0042_auto_20170505_1148.py (hunk 2 above)
Comment (richtier) [inline company/migrations/0042_auto_20170505_1148.py]: its used in the `reverse` argument of the migration. Without it django throws an exception when reverse migration happens.

we want reverse migrations to run to facilitate testing migrations.

File: company/search.py (hunk 3 above)
Comment (richtier) [inline company/search.py]: I think both work equally well. Feel free to change it in a follow up PR if you're passionate about the change.

**Pull request merged**